_includes/css/agency.css

```
@@ -4,6 +4,10 @@
  * For details, see http://www.apache.org/licenses/LICENSE-2.0.
  */
 
+th, td {
+  padding: 20px;
```

```suggestion
  padding: 10px;
```
Reduce padding to 10.